WebsocketClientTransport is incompatible with browser environments

Attempting to compile `WebsocketClientTransport` (for example w/ Webpack) from `@rsocket/rsocket-websocket-client` results in an exception.

```
../../../rsocket-js/node_modules/ws/lib/sender.js:5:0
Module not found: Can't resolve 'net'

Import trace for requested module:
./..\..\..\rsocket-js\node_modules\ws\index.js
./..\..\..\rsocket-js\packages\rsocket-websocket-client\dist\WebsocketClientTransport.js
./..\..\..\rsocket-js\packages\rsocket-websocket-client\dist\index.js
./pages\index.js
```

In the current `0.0.26` version of `rsocket-websocket-client` the `WebSocket` implementation defaults to the global `WebSocket` available in browsers. `1.0.0` has since been changed to default to `WebSocket` made available by the `ws` module. We should likely revert this behavior to again default to browser compatibility, with non-browser compatibility provided via `wsCreator`.

--- a/packages/rsocket-websocket-client/package.json
+++ b/packages/rsocket-websocket-client/package.json
@@ -17,7 +17,6 @@
     "@rsocket/rsocket-core": "^1.0.0"
   },
   "devDependencies": {
-    "@types/ws": "^7.4.7",
     "rimraf": "~3.0.2",
     "typescript": "~4.3.0"
   }

--- a/packages/rsocket-websocket-client/src/WebsocketClientTransport.ts
+++ b/packages/rsocket-websocket-client/src/WebsocketClientTransport.ts
@@ -3,7 +3,6 @@ import {
   Deserializer,
   DuplexConnection,
 } from "@rsocket/rsocket-core";
-import WebSocket, { ErrorEvent } from "ws";
 import { WebsocketDuplexConnection } from "./WebsocketDuplexConnection";
 
 export type ClientOptions = {

--- a/packages/rsocket-websocket-client/src/WebsocketDuplexConnection.ts
+++ b/packages/rsocket-websocket-client/src/WebsocketDuplexConnection.ts
@@ -6,7 +6,6 @@ import {
   serializeFrame,
   StreamIdGenerator,
 } from "@rsocket/rsocket-core";
-import WebSocket, { CloseEvent, ErrorEvent } from "ws";
 
 export class WebsocketDuplexConnection
   extends ClientServerInputMultiplexerDemultiplexer

--- a/packages/rsocket-websocket-client/src/__tests__/WebsocketDuplexConnection.spec.ts
+++ b/packages/rsocket-websocket-client/src/__tests__/WebsocketDuplexConnection.spec.ts
@@ -1,4 +1,3 @@
-import WebSocket from "ws";
 import { mock } from "jest-mock-extended";
 import {
   Deserializer,

